client/src/components/Home.js

```diff
@@ -1,112 +1,105 @@
 import React, { Component } from 'react';
-import { Divider } from 'semantic-ui-react';
+import { Image, Divider, Container } from 'semantic-ui-react';
 import { connect } from 'react-redux';
 import * as Auctions from '../actions/auctionActionCreator.jsx';
 import * as Artists from '../actions/artistActionCreator.jsx';
-//only before frontend connects with backend
-import { artworks } from '../../../server/database/dummyData.js';
 
-console.log('artworks', artworks);
+
+const clickArt = (artId, history) => {
+  history.push('/auction/' + artId);
+};
+
+//render the description ... as floating right of the image
+//closed auctions should be rendered differently from the ongoing ones
+let MainArt = ({art, history}) => {
+  console.log('heyhey:' , art.first_name);
+  return (
+    <span>
+      <Image className="ui image" src={art.artwork.image_url} onClick={() => {
+        clickArt(art.artwork.id, history);
+      }} />
+      <span className="ui label">
+        <span>Artist: {art.first_name} {art.last_name}</span>
+        <br />
+        <span> Description: {art.artwork.description}</span>
+        <br />
+        <span> Year: {art.artwork.age}</span>
+        <br />
+        <span> Closing price: missing!!!!</span>
+      </span>
+    </span>
+  );
+}
+
 let MainArts = ({mainArts, history}) => {
```

Comment:
your destructuring should follow a standard protocol. in your imports you did { property }. here you do {property};